package/crds/s3.aws.crossplane.io_buckets.yaml

```diff
@@ -885,7 +885,6 @@ spec:
                             type: string
                         required:
                         - hostName
-                        - protocol
```

Comment:
CRDs are automatically generated. You'll need to make [this field](https://github.com/crossplane/provider-aws/blob/master/apis/s3/v1beta1/websiteConfiguration_types.go#L64) 
```
Protocol *string `json:"protocol,omitempty"`
```
 to make it non-required then fix where it's used.

Reply:
How do i regen these files? Since i have made a change to the overall structs

Reply:
you can run `make reviewable`

Reply:
Running now, will push once its done (i forgot about that command as i ran `test` and `e2e` only 🤦 )

Reply:
Ran it locally, no difference in output (assuming this means my hand-crafted changes are correct)